amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2862,6 +2968,12 @@ parameters:
     description: The identifier of a child of a hold.
     required: true
     type: string
+  processId:
+    name: processId
+    in: path
+    description: The identifier of a bulk process.
+    required: true
+    type: string
```

Comment:
Why did we need to add this here and why is it `required: true`?

Won't this break existing APIs such as https://github.com/Alfresco/alfresco-community-repo/blob/f19cc7f446db43ae1a41a85d17fe49edc767495a/amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml#L2575-L2585 if you don't pass the `processId`?

Reply:
We need this path parameter for an endpoint to get a single process by id. It is required to be set. I don't think it will break existing APIs since the "_Delete a child of a hold_" endpoint doesn't have the **processId** parameter set. Moreover, I don't see any path parameter in this API that doesn't have a **required** clause. 
See the Nodes API: https://api-explorer.alfresco.com/api-explorer/#/nodes

Reply:
Nevermind, all good. I mistakenly saw it as if it was nested under https://github.com/Alfresco/alfresco-community-repo/pull/2624/files#diff-59438a0c7441d161bfdee3ac2910ca386af01a40dac46cd94c42506918cb10f5R2965.